sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/implementation/SyncTokenPolicyTest.java

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.implementation;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.http.MockHttpResponse;
+import com.azure.core.test.http.NoOpHttpClient;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SyncTokenPolicyTest extends TestBase {
```

Comment:
This doesn't need to inherit from TestBase. You aren't actually making any playback/record tests. It'll decrease the need for those empty *.json files.

Reply:
I remembered when I have no *.json file for new tests, it failed on PR pipeline complaining there is no json file for it. So a unit test will be fine without .json files? I will check it later but want to have a hint from you as well. Thanks

Reply:
The Event Hubs tests don't have *.json files. We also don't extend from TestBase.

Reply:
The pipeline fails because the setup step in TestBase looks for a .json file.